authentik/sources/oauth/models.py

```diff
@@ -65,9 +67,8 @@ def type(self) -> type["SourceType"]:
     def component(self) -> str:
         return "ak-source-oauth-form"
 
-    # we're using Type[] instead of type[] here since type[] interferes with the property above
     @property
-    def serializer(self) -> Type[Serializer]:
+    def serializer(self) -> SerializerType:
```

Comment:
for this case we should probably just rename the other type attribute on the model since iirc it's just a property, but we can do that later

Reply:
Yes, that would be great to remove this workaround 👍

Reply:
Will create an issue to keep track of this